doc/api/crypto.md

```diff
@@ -3895,6 +3895,130 @@ Type: {Crypto} An implementation of the Web Crypto API standard.
+
+The `algorithm` is dependent on the available algorithms supported by the
+version of OpenSSL on the platform. Examples are `'sha256'`, `'sha512'`, etc.
+On recent releases of OpenSSL, `openssl list -digest-algorithms`
+(`openssl list-message-digest-algorithms` for older versions of OpenSSL) will
+display the available digest algorithms.
+
+### `cryptoPromises.sign(algorithm, data, key[, options])`
+<!-- YAML
+added: REPLACEME
+-->
+
+* `algorithm` {string|null|undefined}
+* `data` {ArrayBuffer|TypedArray|DataView|Buffer}
+* `key` {KeyObject|CryptoKey}
+* `options` {Object}
+  * `dsaEncoding` {string} For DSA and ECDSA, this option specifies the
+    format of the generated signature. It can be one of the following:
+    * `'der'` (default): DER-encoded ASN.1 signature structure encoding `(r, s)`.
+    * `'ieee-p1363'`: Signature format `r || s` as proposed in IEEE-P1363.
+  * `padding` {integer} Optional padding value for RSA, one of the following:
+    * `crypto.constants.RSA_PKCS1_PADDING` (default)
+    * `crypto.constants.RSA_PKCS1_PSS_PADDING`
+  * `saltLength` {integer} Salt length for when padding is
+    `RSA_PKCS1_PSS_PADDING`. The special value
+    `crypto.constants.RSA_PSS_SALTLEN_DIGEST` sets the salt length to the digest
+    size, `crypto.constants.RSA_PSS_SALTLEN_MAX_SIGN` (default) sets it to the
+    maximum permissible value.
+* Returns: {Promise} containing {Buffer}
+
+Calculates the signature for `data` using the given private key and
+algorithm. If `algorithm` is `null` or `undefined`, then the algorithm is
+dependent upon the key type (especially Ed25519 and Ed448).
+
+### `cryptoPromises.verify(algorithm, data, key, signature[, options])`
+<!-- YAML
+added: REPLACEME
+-->
+
+* `algorithm` {string|null|undefined}
+* `data` {ArrayBuffer|TypedArray|DataView|Buffer}
+* `key` {KeyObject|CryptoKey}
+* `signature` {ArrayBuffer|TypedArray|DataView|Buffer}
+* `options` {Object}
+  * `dsaEncoding` {string} For DSA and ECDSA, this option specifies the
+    format of the generated signature. It can be one of the following:
+    * `'der'` (default): DER-encoded ASN.1 signature structure encoding `(r, s)`.
+    * `'ieee-p1363'`: Signature format `r || s` as proposed in IEEE-P1363.
+  * `padding` {integer} Optional padding value for RSA, one of the following:
+    * `crypto.constants.RSA_PKCS1_PADDING` (default)
+    * `crypto.constants.RSA_PKCS1_PSS_PADDING`
+  * `saltLength` {integer} Salt length for when padding is
+    `RSA_PKCS1_PSS_PADDING`. The special value
+    `crypto.constants.RSA_PSS_SALTLEN_DIGEST` sets the salt length to the digest
+    size, `crypto.constants.RSA_PSS_SALTLEN_MAX_SIGN` (default) sets it to the
+    maximum permissible value.
+* Returns: {Promise} containing {boolean}
+
+Verifies the given signature for `data` using the given key and algorithm. If
+`algorithm` is `null` or `undefined`, then the algorithm is dependent upon the
+key type (especially Ed25519 and Ed448).
```

Comment:
likewise here, move the details about the `algorithm` default into the args bullet this.